ttnmqtt/ttnmqtt.py

```diff
@@ -0,0 +1,203 @@
+# coding: Latin-1
+# Copyright © 2017 The Things Network
+# Use of this source code is governed by the
+# MIT license that can be found in the LICENSE file.
+
+
+import paho.mqtt.client as mqtt
+from events import Events
+import json
+import base64
+from collections import namedtuple
+from .gRPCfiles import discovery_pb2_grpc
+from .gRPCfiles import discovery_pb2
+import grpc
+import os
+
+# Necessary to make gRPC work
+if os.getenv('GRPC_SSL_CIPHER_SUITES'):
+    os.environ['GRPC_SSL_CIPHER_SUITES'] += os.pathsep + os.pathsep.join(
```

Comment:
Avoid duplicating the list of cipher suites in code by extracting it to a separate variable.